conf/application.conf

```diff
@@ -335,6 +335,7 @@ urls {
     paye = "/paye-online"
     VATRegistration = "/vat-registration/how-to-register#when-you-cannot-register-online"
     onlineServicesHelpdesk = "/government/organisations/hm-revenue-customs/contact/online-services-helpdesk"
+    checkVATNumber = "/check-uk-vat-number"
```

Comment:
May I know why is this required?

Reply:
please note, all gov links in views are added in the app config.  In this case, we have one for this text on the interstitial page.
"Check a UK VAT Number service (opens in new tab)"
Easy to change in future.